src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,138 @@
+import React = require('react');
+import {EventHandler, ReactElement, SyntheticEvent} from "react";
+import {observable, action} from 'mobx';
+import {observer} from 'mobx-react';
+const style = require('./radio-group.css');
+
+export interface RadioGroupProps {
+    children: JSX.Element[];
+    onChange: any;
+    location?: "right" | "left";
+    name?: string;
+}
+
+let counter = 0;
+
+export interface RadioState {
+    checked: boolean;
+}
+
+@observer
+export class RadioGroup extends React.Component<RadioGroupProps, {}> {
+    name: string;
+    checkedArray: Array<RadioState>;
```

Comment:
If I understand it right, `input type="radio"` suggests exactly one selected value in the group, so why array?